src/lib/apputils/net-server.c

```diff
@@ -753,7 +764,7 @@ setup_socket(struct socksetup *data, struct bind_address *ba,
     }
 
     /* Try to turn on pktinfo for UDP wildcard sockets. */
-    if (ba->type == UDP && ba->address == NULL) {
+    if (ba->type == UDP && is_wildcard(sock_address)) {
         krb5_klog_syslog(LOG_DEBUG, _("Setting pktinfo on socket %s"),
```

Comment:
I can't recall the exact reason right now but there was definitely a reason why I didn't use pktinfo for all addresses, just wildcard. I'll have to look into it when I have access to my laptop.